docs/src/GettingStarted/BecomeProvider.md

```diff
@@ -0,0 +1,54 @@
+# Become a Provider
+
+A Provider is the on-chain representation of an application.
+Any application that will interact with users must first register as a Provider.
+This guide will walk you through the steps to becoming a Provider on the Frequency Testnet. See [How to Become a Provider on Mainnet](#mainnet), if you are ready to move to production.
```

Comment:
I'm not sure I agree with this description. Technically, a Provider represents an entity that wishes to run one **OR MORE** applications providing Frequency services to users.

Example:
Meta is a organization that runs the Instagram and Facebook applications
Meta could provision a single Provider and run both apps under it, or it could provision a separate Provider for each app.

Really, the decision of an organization of how to partition their applications amongst one or more registered Providers is a business decision that takes into account the economic model & how the organization wishes to leverage Capacity.

Reply:
Agreed, but not sure how to better state that for a getting started guide. Open to ideas.

Reply:
```suggestion
A Provider is a special kind of user account on Frequency, capable of executing certain operations on behalf of other users (delegators).
Any organization wishing to deploy an application that will act on behalf of users must first register as a Provider.
This guide will walk you through the steps to becoming a Provider on the Frequency Testnet. See [How to Become a Provider on Mainnet](#mainnet), if you are ready to move to production.
```

Reply:
Works for me. Committed.